site/static/examples/jobs/sample-pytorchjob.yaml

```diff
@@ -14,8 +14,10 @@ spec:
         spec:
           containers:
             - name: pytorch
-              image: docker.io/kubeflowkatib/pytorch-mnist:v1beta1-45c5727
-              imagePullPolicy: Always
+              image: docker.io/kubeflowkatib/pytorch-mnist-gpu:latest
```

Comment:
Could you specify the specific tag? The latest tag sometimes doesn't work well since the latest tag is developing image.